.eslintrc

```diff
@@ -76,10 +76,11 @@
         "camelcase": 2,
         "comma-spacing": 2,
         "comma-style": [2, "last"],
+        "comma-dangle": [2, "always-multiline"],
         "computed-property-spacing": [2, "never"],
         "consistent-this": [2, "self"],
         "func-names": 2,
-        "func-style": [2, "expression"],
+        "func-style": [2, "declaration", { "allowArrowFunctions": true }],
```

Comment:
This allows us to use normal functions when exporting a module

```
export default function nameOfFunction () {
```

And arrow functions when declaring a variable:

```
let someFunction = () => {
```

In my opinion, that's how it should be.